src/repositories/DocumentPageRepository.ts

```diff
@@ -1,21 +1,21 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import type { DocumentPageApi, FrontifyDocumentPage } from '../types';
+import type { DocumentPage, DocumentPageApi } from '../types';
 import { HttpClient } from '../utilities/httpClient';
 
-export const getDocumentPagesByDocumentId = async (documentId: number): Promise<FrontifyDocumentPage[]> => {
+export const getDocumentPagesByDocumentId = async (documentId: number): Promise<DocumentPage[]> => {
     const { result } = await HttpClient.get<DocumentPageApi[]>(`/api/document-page?document_id=${documentId}`);
     return result.data.map(mapDocumentPageApiToFrontifyDocumentPage);
 };
 
-export const getUncategorizedPagesByDocumentId = async (documentId: number): Promise<FrontifyDocumentPage[]> => {
+export const getUncategorizedPagesByDocumentId = async (documentId: number): Promise<DocumentPage[]> => {
     const { result } = await HttpClient.get<DocumentPageApi[]>(
         `/api/document-page?document_id=${documentId}&exclude_categorized`,
     );
     return result.data.map(mapDocumentPageApiToFrontifyDocumentPage);
 };
 
-export const mapDocumentPageApiToFrontifyDocumentPage = (documentPageApi: DocumentPageApi): FrontifyDocumentPage => ({
+export const mapDocumentPageApiToFrontifyDocumentPage = (documentPageApi: DocumentPageApi): DocumentPage => ({
```

Comment:
Should update method name as well:
```suggestion
export const mapDocumentPageApiToDocumentPage = (documentPageApi: DocumentPageApi): DocumentPage => ({
```